ldap/servers/slapd/main.c

```diff
@@ -1062,6 +1062,8 @@ main(int argc, char **argv)
         eq_start(); /* must be done after plugins started - DEPRECATED */
         eq_start_rel(); /* must be done after plugins started */
 
+        vattr_check(); /* Check if it exists virtual attribute definitions */
```

Comment:
Would it be worth a call to vattr_check from both roles/cos when a modification occurs? I can see that if the server is started and there are no virtual attrs, then this would be "ignore=on". Then someone adds a role or cos template, and may wonder why it's not taking effect. So in this case, shouldn't we re-check during the mod live? Else they'll need to restart at least once to get this to work.